tests/test_export_item.py

```diff
@@ -397,29 +397,66 @@ def test_display_name():
 
     assert dataio.metadata4display["name"] == "ObjectName"
 
-    # no name, no display_name, no object-name: Return None
+    # no display_name, no name. Use object name. (grid)
     dataio = fmu.dataio.ExportData(
         config=CFG2,
         content="depth",
         verbosity="DEBUG",
     )
 
-    obj = xtgeo.RegularSurface(ncol=3, nrow=4, xinc=22, yinc=22, values=0)
-    exporter = ei._ExportItem(dataio, obj, verbosity="DEBUG")
+    grid = xtgeo.Grid()
+    grid.create_box()
+    grid.name = "ObjectName"
```

Comment:
Prefer grid = xtgeo.create_box_grid() syntax (ongoing change in xtgeo)

Reply:
Done